index.js

```diff
@@ -88,7 +88,7 @@ function fastProxy (opts = {}) {
       }
       request(reqParams, (err, response) => {
         if (err) {
-          if (!res.sent) {
+          if (!res.sent || !res.writableFinished) {
```

Comment:
It might be helpful to have a comment here to explain these two separate attributes of the `res` and why we are checking both.

Reply:
I like the suggestion and indeed, you are correct. It still would have executed that conditional in that case. I can't speak for `res.sent` but I know that `res.writableFinished` is indeed a boolean. I think it would be best to explicitly check with an or in this case in case `res.sent` is conceptually distinct in some way